verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -209,4 +211,17 @@ export default class SourcesController {
         res.status(204).end();
         return;
     };
+
+    /** Trigger retrieval of the source's content in S3. */
+    retrieve = async (req: Request, res: Response): Promise<void> => {
```

Comment:
Worth adding a quick unit test for this?

Reply:
always worth it, done